MTV-614: adding release notes and attribute update for 2.4.2 (#398)
kubev2v/forklift-documentation: documentation/modules/rn-2.4.adoc

This release has the following technical changes:

.Faster disk image migration from {rhv-short}

> Disk images are not converted anymore using virt-v2v when migrating from RHV. This change speeds up migrations and also allows migration for guest operating systems that are not supported by virt-vsv. link:https://github.com/kubev2v/forklift-controller/issues/403[(forklift-controller#403)]

Reviewer: ```suggestion
Disk images are not converted anymore using virt-v2v when migrating from {rhv-short}. This change speeds up migrations and also allows migration for guest operating systems that are not supported by virt-vsv. link:https://github.com/kubev2v/forklift-controller/issues/403[(forklift-controller#403)]
```

Some warm migrations from {rhv-short} might fail. When running a migration plan for warm migration of multiple VMs from {rhv-short}, the migrations of some VMs might fail during the cutover stage. In that case, restart the migration plan and set the cutover time for the VM migrations that failed in the first run. link:https://bugzilla.redhat.com/show_bug.cgi?id=2063531[(BZ#2063531)] link:https://issues.redhat.com/browse/MTV-456[(MTV-456)]

Warm migration from {rhv-short} fails if a snapshot operation is performed on the source VM. If the user performs a snapshot operation on the source VM at the time when a migration snapshot is scheduled, the migration fails instead of waiting for the user’s snapshot operation to finish. link:https://bugzilla.redhat.com/show_bug.cgi?id=2057459[(BZ#2057459)]

.Cannot schedule migrated VM with multiple disks to more than one storage classes of type hostPath

When migrating a VM with multiple disks to more than one storage classes of type hostPath, it may result in a VM that cannot be scheduled. Workaround: It is recommended to use shared storage on the target OCP cluster.

.Deleting migrated VM does not remove PVC and PV

> When removing a VM that was migrated, its persistent volume claims (PVCs)  and physical volumes (PV) are not deleted. Workaround: remove the CDI importer pods and then remove the remaining PVCs and PVs. link:https://issues.redhat.com/browse/MTV-492[(MTV-492)]

Reviewer: There is a redundant space
```suggestion
When removing a VM that was migrated, its persistent volume claims (PVCs) and physical volumes (PV) are not deleted. Workaround: remove the CDI importer pods and then remove the remaining PVCs and PVs. link:https://issues.redhat.com/browse/MTV-492[(MTV-492)]
```

.Prevent locking user accounts due to incorrect credentials

If a user specifies an incorrect password for {rhv-short} providers, they are no longer locked in {rhv-short}. An error returns when the {rhv-short} manager is accessible and adding the provider. If the {rhv-short} manager is inaccessible, the provider is added, but there would be no further attempt after failing, due to incorrect credentials. link:https://issues.redhat.com/browse/MTV-324[(MTV-324)]

.Users without cluster-admin role can create new providers

> Previously, the `cluster-admin` role was required to browse and create providers. In this release, users with sufficient permissions on MTV resources (providers, plans, migrations, NetworkMaps, storage maps, hooks) can operate MTV without cluster-admin permissions. link:https://issues.redhat.com/browse/MTV-334[(MTV-334)]

Reviewer 2: in this case we should also change storage maps:
```suggestion
Previously, the `cluster-admin` role was required to browse and create providers. In this release, users with sufficient permissions on MTV resources (providers, plans, migrations, NetworkMaps, StorageMaps, hooks) can operate MTV without cluster-admin permissions. link:https://issues.redhat.com/browse/MTV-334[(MTV-334)]
```

.Convert i440fx to q35

Migration of virtual machines with i440fx chipset is now supported. The chipset is converted to q35 during the migration. link:https://issues.redhat.com/browse/MTV-430[(MTV-430)]

.Preserve the UUID setting in SMBIOS for a VM that is migrated from {rhv-short}

The Universal Unique ID (UUID) number within the System Management BIOS (SMBIOS) no longer changes for VMs that are migrated from {rhv-short}. This enhancement enables applications that operate within the guest operating system and rely on this setting, such as for licensing purposes, to operate on the target OCP cluster in a manner similar to that of {rhv-short}. link:https://issues.redhat.com/browse/MTV-597[(MTV-597)]

.Do not expose password for {rhv-short} in error messages

> Previously, the password that was specified for ovirt-engine appeared in error messages that were displayed in the web console and logs when failing to connect to {rhv-short}. In this release, error messages that are generated when failing to connect to RHV do not reveal the password for ovirt-engine.

Reviewer: ```suggestion
Previously, the password that was specified for ovirt-engine appeared in error messages that were displayed in the web console and logs when failing to connect to {rhv-short}. In this release, error messages that are generated when failing to connect to {rhv-short} do not reveal the password for ovirt-engine.
```

> Previously, the password that was specified for ovirt-engine appeared in error messages that were displayed in the web console and logs when failing to connect to {rhv-short}. In this release, error messages that are generated when failing to connect to {rhv-short} do not reveal the password for ovirt-engine.

Reviewer 2: @RichardHoch suggested to use the downstream term for RHV here instead of ovirt-engine, which makes sense. for consistency we can use RHV manager as we do elsewhere in the release notes
```suggestion
Previously, the password that was specified for {rhv-short} manager appeared in error messages that were displayed in the web console and logs when failing to connect to {rhv-short}. In this release, error messages that are generated when failing to connect to {rhv-short} do not reveal the password for {rhv-short} manager.
```
